### ISSUE TITLE
JSDoc link in Documentation is broken

### ISSUE THREAD

**Issue by jnegro** [gitana/alpaca#512]
http://code.cloudcms.com/alpaca/1.5.23/jsdoc/index.html results in an XML error page:

<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>alpaca/1.5.23/jsdoc/index.html</Key><RequestId>4BD23F3E4A6A474D</RequestId><HostId>CtmiMMBEwh3zBJMY0EoHM3NEAySh667HNrcZP19rc5DKv/IvCIGlp/Ukr0CNp3reGFT2UD1gamY=</HostId></Error>

I'm hoping this will give me a better view into the low-level API.

**Comment by LukeTOBrien** [comment]
I am receiving the following XML:

``` XML
<Error>
<Code>AccessDenied</Code>
<Message>Access Denied</Message>
<RequestId>6C65FA86BA55DADA</RequestId>
<HostId>
ao96NKfVOzkPtnKb9ytwWwu6B0zksqVo+wq7ldNoICuzmGGMRMKryPmlHfLmp27IN8fIwjU0fuc=
</HostId>
</Error>
```

**Comment by codek** [comment]
Yup i'm seeing this too, no ideas? I can't find the jsdoc in the master zip either? But surely it's there?
